actions/status-checker/src/pull-updater.ts

```diff
@@ -12,7 +11,7 @@ const PREVIEW_TABLE_END = '<!-- PREVIEW-TABLE-END -->';
 export async function tryUpdatePullRequestBody(token: string) {
   try {
     const prNumber: number = context.payload.number;
-    startGroup(`Update pull ${prNumber} request body.`);
+    console.log(`Update pull ${prNumber} request body.`);
```

Comment:
```suggestion
    console.log("Update pull ${prNumber} request body.");
```

Reply:
This would break the logging. The backtick in JS is for string interpolation.